xml/obs_ag_spider_identification.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter PUBLIC
+  "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.docbook.org/xml/4.5/docbookx.dtd"
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+  %entities;
+]>
+<section id="_obs_spider_identification">
+ <title>Spider Identification</title>
+ <para>
+   OBS identifys spiders by using the gem <command>voight_kampff</command>. Spiders will be identified to optimize the
+   page content for them.
+ </para>
+ <para>
+   To update the spider list, you must run the command <command>rake voight_kampf:import_user_agents</command>.
+   This downloads a list of spiders as a JSON file into the <filename>config/</filename> directory of the Rails application.
+ </para>
+ <para>
+   Switch to the <filename>config/</filename> directory and open the <filename>crawler-user-agents.json</filename> file with
```

Comment:
Hmm, I'm missing here the part that this is only necessary if the bot is not included in the list / if it is a custom bot. And furthermore, I would prefer not to maintain a custom list for this and instead follow the recommended way and a send a PR to https://github.com/monperrus/crawler-user-agents. Like it is recommended in the [gem readme](https://github.com/biola/Voight-Kampff).